databricks/sdk/core.py

```diff
@@ -882,9 +894,11 @@ def _record_request_log(self, response: requests.Response):
             for k, v in request.headers.items():
                 sb.append(f'> * {k}: {self._only_n_bytes(v, self._debug_truncate_bytes)}')
         if request.body:
-            sb.append(self._redacted_dump("> ", request.body))
+            sb.append("> [raw stream]" if raw else self._redacted_dump("> ", request.body))
         sb.append(f'< {response.status_code} {response.reason}')
-        if response.content:
+        if raw and 'Content-Type' in response.headers:
```

Comment:
Why check for `Content-Type` header here?

Reply:
Raw streams with `Transfer-Encoding: chunked` do not have `Content-Type` header